Ember/app/styles/footer-component.scss

```diff
@@ -0,0 +1,25 @@
+.footer {
+    bottom: 0;
+    width: 100%;
+    position: absolute;
+    height: 4em;
+    background-color:#332e2e;
+
+}
+
+.list-inline-item{
+    color:white!important;
```

Comment:
Kad moraš koristiti `!important` znaš da nešto nije uredu. U dobro napisanom CSS-u ne bi nikad trebalo da se koristi `!important`